app/web/features/search/SearchBox.tsx

```diff
@@ -118,25 +119,29 @@ export default function SearchBox({
     setValue("query", searchFilters.active.query ?? "");
   }, [setValue, searchFilters.active.location, searchFilters.active.query]);
 
-  const filterDialogButton = (
-    <>
-      <Button
-        onClick={() => setIsFiltersOpen(true)}
-        className={classNames(className, classes.filterDialogButton)}
-        variant={isSmDown ? "contained" : "outlined"}
-        size="small"
-      >
-        {FILTER_DIALOG_TITLE}
-      </Button>
-      <FilterDialog
-        isOpen={isFiltersOpen}
-        onClose={() => setIsFiltersOpen(false)}
-        searchFilters={searchFilters}
-      />
-    </>
+  const filterDialog = (
+    <FilterDialog
+      isOpen={isFiltersOpen}
+      onClose={() => setIsFiltersOpen(false)}
+      searchFilters={searchFilters}
+    />
   );
+
   if (isSmDown) {
-    return filterDialogButton;
+    return (
+      <>
+        <Button
+          onClick={() => setIsFiltersOpen(true)}
+          className={classNames(className)}
```

Comment:
```suggestion
          className={className}
```
No need to use the `classNames` function if `className` is the only list of classes that will be applied here?

Reply:
of course, thanks!